src/magic.cpp

```diff
@@ -217,6 +218,12 @@ void spell_type::load( JsonObject &jo, const std::string & )
     mandatory( jo, was_loaded, "name", name );
     mandatory( jo, was_loaded, "description", description );
     optional( jo, was_loaded, "message", message, to_translation( "You cast %s!" ) );
+    optional( jo, was_loaded, "sound_description", sound_description,
```

Comment:
You need to update `extract_json_strings.py` to extract this string for translation.